### ISSUE TITLE
Showing tail is sometimes redundant, need a way to turn it off

### ISSUE THREAD

**Issue by Suseika** [yegor256/rultor#881]
#807 introduces a great feature of showing the tail of build log if something fails. But sometimes the tails of failed build attemts flood the conversation in an issue, if build is failed repeatedly. 

Can we have a feature to turn showing tail off, leaving just the "Oops, I failed!" message with the link to the full log? For example, `quiet` or `notail` command in a message to rultor, or a directive in `.rultor.yml` that disables showing tails.

**Comment by krzyk** [comment]
@Suseika I don't think that it is a good idea, we introduced tail because it is easier to look at those and not click on a link and scroll down a MB of text.
It is hard to decide when this tail could be removed, it floods the conversation only if it is repeating what was in the previous failed build, so maybe add a comparison if the previous tail is the same as the current one then don't show it.

**Comment by Suseika** [comment]
@krzyk 
> It is hard to decide

What do you mean by decide? If I want the tail, I say `rultor merge`; if I don't want it, I say `rultor merge quiet`. By default tails will still be there, as I see it.

**Comment by krzyk** [comment]
@Suseika sorry I saw only the `.rultory.yml` part, yep, adding `quiet` is a good idea

**Comment by Suseika** [comment]
@krzyk Yeah, the `.rultor.yml` suggestion was probably not a good one, it's quite implicit.

**Comment by yegor256** [comment]
@Suseika I don't really like this `quiet` idea... it's making the interface more complex and less intuitive... what exactly is the problem? you don't like to see these long listings in Github messages? but they are not very long..

**Comment by Suseika** [comment]
@yegor256 Logs are not that long, but sometimes you expect many consecutive builds to fail, so a length of a log multiplied by a number of attempts becomes quite significant. I had a problem when I was trying out rultor in one of my repositories. I expected builds to fail (and they did) because it was the first  time I tried to configure rultor to release a new version: Suseika/rocollections#4. Basically, in an "it works for me but doesn't work for rultor" situation when I expect a lot of failed builds I'd like to have that feature.

**Comment by yegor256** [comment]
@Suseika @krzyk how about another idea. When rultor posts a new tail it goes up through all previous messages and deletes all tails there. Thus, we will always have the last tail only. What do you think?

**Comment by Suseika** [comment]
@yegor256 That would be great for me, but I'm not sure if others using rultor would need the old tails or not.

**Comment by krzyk** [comment]
@yegor256 I think the best would be to put it in spoiler tag (click to show)  but github doesn't seem to have one. 

I wouldn't go with removing tails,  unless it is the same as the last one. Sometimes it is useful to find when certain failures started appearing.

**Comment by yegor256** [comment]
@krzyk yeah... indeed, deleting history is a rather bad practice... let's keep this ticket on hold and see what other users may think about it

**Comment by yegor256** [comment]
@alex-palevsky let's postpone it

**Comment by alex-palevsky** [comment]
@suseika we will find someone to do this task, soon

**Comment by alex-palevsky** [comment]
@suseika I added milestone `2.0` to this issue, let me know if there has to be something else

**Comment by alex-palevsky** [comment]
@suseika thanks for reporting! I topped your account for 15 mins, transaction AP-67T38338FX960393B

**Comment by alex-palevsky** [comment]
> @alex-palevsky let's postpone it

@yegor256 sure, thanks, I added "postponed" tag to it

**Comment by alex-palevsky** [comment]
> @alex-palevsky let's postpone it

@yegor256 I will try to find someone else
